less/mixins/grid.less

```diff
@@ -13,8 +13,10 @@
 
 // Creates a wrapper for a series of columns
 .make-row(@gutter: @grid-gutter-width) {
-  margin-left:  ceil((@gutter / -2));
-  margin-right: floor((@gutter / -2));
+  @_margin_left_helpvar:  ceil((@gutter / 2));
```

Comment:
Don't bother using variables for these, just put them directly into the property values.